packages/make-styles/README.md

```diff
@@ -76,6 +76,51 @@ const masks = {
 
 It also made code complicated for understanding and debugging.
 
+### How about static styles?
+
+There is a separate API `makeStaticStyles` for this case.
+
+It can be used to register styles object:
+
+```jsx=
+makeStaticStyles({
+  '@font-face': {
+    fontFamily: 'Open Sans',
+    src: `url("/fonts/OpenSans-Regular-webfont.woff2") format("woff2"),
+         url("/fonts/OpenSans-Regular-webfont.woff") format("woff")`,
+  },
+  body: {
+    background: 'red',
+  },
```

Comment:
```suggestion
  },
  // ⚠️ nested and preudo selectors are not supported for this scenario via nesting
  '.some.component:hover': { color: 'green' }
```

I think that we should mention this, if you have a better description - go with it 🐱